website/docs/artist_hosts_houdini.md

```diff
@@ -83,6 +83,24 @@ select your render camera.
 All the render outputs are stored in the pyblish/render directory within your project path.\
 For Karma-specific render, it also outputs the USD render as default.
 
+## Publishing cache to Deadline
+Five Instance (Arnold ASS, Pointcache(BGEO and ABC) and Redshift Proxy) are supported for
```

Comment:
I can only count 4?
.ass, .bgeo, .abc and whatever redshit proxy extension is?

Reply:
.ass, .bgeo, .abc, .rsproxy, and .vdb. I will update on that

Reply:
I feel like this is also better written as a clear list, e.g.

---

Caching on the farm is supported for:

- Arnold ASS
- Pointcache (BGEO and ABC)
- Redshift Proxy